src/main/kotlin/ktee/KTee.kt

```diff
@@ -1,7 +1,9 @@
 package ktee
 
+import ktee.KTee.Companion.debug
 import org.slf4j.Logger
 
+class KTee { companion object { var debug = true } }
```

Comment:
we actively avoid using `var` in our codebase. This becomes a global variable which is not good in server/multithreaded/concurrent envs.

Reply:
I've made some changes. The variable can now be set only once.
If you are concerned about thread safety, @Volatile can also be used instead.
Other than that, I fail to see how it's possible to have a toggle without being a var.